solr/core/src/java/org/apache/solr/core/SolrCore.java

```diff
@@ -3013,10 +3013,6 @@ public static void postDecorateResponse(
     }
   }
 
-  public static final void log(Throwable e) {
-    SolrException.log(log, null, e);
-  }
```

Comment:
I can leave this to cleanup w/ the other SolrException.log, but this was only used in 3 places so just cleaned it up.